python/paddle/fluid/distributed/ps_pb2.py

```diff
@@ -0,0 +1,1506 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Please don't add the pb2 file into the repo, maybe compile the protobuf by cmake is a better way, for example: https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/framework/CMakeLists.txt#L36 .

Reply:
the protobuf is related to pslib, now, don't want to exposure of this.